CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to `wasmtime-py`
+
+`wasmtime-py` is a [Bytecode Alliance] project. It follows the Bytecode
+Alliance's [Code of Conduct] and [Organizational Code of Conduct].
+
+So far this extension has been written by folks who are primarily Rust
+programmers, so it's highly likely that there's some faux pas in terms of Python
+idioms. Feel free to create a PR to help make things more idiomatic if you see
+something!
```

Comment:
I'm pretty sure that by this point, this extension does qualify as "idiomatic Python"!

Reply:
Heh yeah I'm happy to remove this clause if its unnecessary now (I'm also not really in a position to judge whether this package is idiomatic or not :sweat_smile: )

Reply:
I'm sure there are a few things that could be further improved, but Python packages aren't all that consistent in general. Wasmtime-py seems at the very least better than average to me, which means the warning no longer applies!